src/css/Support.css

```diff
@@ -0,0 +1,32 @@
+.Support-Text{
+    color: black;
+    text-align: left;
+    opacity: 1;
+    font: Regular 25vh/30vh Whyte Inktrap;
```

Comment:
This doesn't seem to work on my machine, do separate font-family and font size css rules to accomplish the same thing instead. Font should be Lato and font-size should be around 2.2vh so that it resembles the 20px font size from the ref.